src/main/java/BST/CommonProblems/MirrorBST.java

```diff
@@ -0,0 +1,24 @@
+package BST.CommonProblems;
+
+import BST.TreeNode;
+
+/**
+ * Created by sougata on 24/08/16.
+ */
+public class MirrorBST {
+
+    public TreeNode createMirror(final TreeNode root) {
+
+        if(root == null)
+            return null;
+
+        if(root!=null){
```

Comment:
if (root != null) { ..... }
return root

You will return null if root is null anyways. Please fix.